src/onboarding/containers/LoginPage.tsx

```diff
@@ -63,11 +56,21 @@ export const LoginPage: FC = () => {
   } else {
     if (isFlagEnabled('universalLogin')) {
       if (CLOUD) {
-        getQuartzLoginUrl({})
+        fetch('/api/env/quartz-login-url')
```

Comment:
will have to follow up on how to add this uiproxyd route to openapi later.

Reply:
I don't think you'll be able to. But I can double check

Reply:
Got it yeah! I tried some things, it didnt really work so I thought we ship this for now. 👍